dev/breeze/doc/09_release_management_tasks.rst

```diff
@@ -788,6 +788,29 @@ default is to build ``both`` type of distributions ``sdist`` and ``wheel``.
   :width: 100%
   :alt: Breeze release-management prepare-airflow-ctl-distributions
 
+Preparing Airflow CTL Tarball
```

Comment:
Should we just call it `airflowctl`? I thought I commented it somewhere but can't remember or I am mis-remembering :)

Reply:
I have updated all in the other PR :) Seems this PR may not even be needed from your next comment :) Let's continue discussing this change there
Original Comment PR: https://github.com/apache/airflow/pull/54683